components/infobox/commons/infobox.lua

```diff
@@ -58,11 +59,11 @@ function Infobox:widgetInjector(injector)
 	return self
 end
 
----Adds a custom widgets to the bottom of the infobox
+---Adds a custom components after the end the infobox
 ---@param wikitext string|number|Html|nil
 ---@return self
 function Infobox:bottom(wikitext)
-	self.bottomContent = wikitext
+	self.bottomContent = (self.bottomContent or '') .. (wikitext or '')
```

Comment:
```suggestion
	self.bottomContent = (self.bottomContent or '') .. (wikitext or '')
	if Logic.isEmpty(wikitext) then
		return self
	end

	self.bottomContent = (self.bottomContent or mw.html.create()):node(wikitext)
```

Reply:
Is the first line suppose to be gone? Otherwise makes little sense

Reply:
yeah